pyproject.toml

```diff
@@ -1,10 +1,84 @@
 [build-system]
-requires = ['setuptools >= 40.9.0', 'wheel', 'packutil']
-build-backend = 'setuptools.build_meta'
+requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+name = "eko"
+version = "0.0.0"
+description = "Evolution Kernel Operator"
+authors = [
+  "A. Candido <alessandro.candido@mi.infn.it>",
+  "S. Carrazza <stefano.carrazza@cern.ch>",
+  "J. Cruz-Martinez",
+  "F. Hekhorn",
```

Comment:
Could you please add the other mails? Do we want to drop SC and JCM?

Reply:
I've been lazy, and since I didn't want to make decisions on which email to put I left it for you guys :)
Add your own, Juan and Giacomo can do the same.

Reply:
@giacomomagni I added your Nikhef mail (as in yadism) - is that alright?

Reply:
perfect, thanks @felixhekhorn :)